README.md

```diff
@@ -90,12 +96,12 @@ Run the containers:
 
     docker-compose up django
 
-Visit http://localhost:8000/petition/api/ in your browser.
+Visit http://localhost:8000/petition/api/ in your browser.  If you get authentication errors, you may login as the superuser you created at http://localhost:8000/ and try again.
 
 ### Using docker-compose.override.yml
 
-To develop in a Docker container, we'll use a `docker-compose.override.yml`
-override file to configure the Django container to sleep by default:
+To develop in a Docker container, we'll create a `docker-compose.override.yml`
+override file in the root of the dear-petition directory to configure the Django container to sleep by default:
```

Comment:
Your changes look good. I think this section could also probably be elaborated upon because it is currently portrayed as if it's a required step. Some ideas to de-prioritize this section:

* Add `(Optional)` to the header
* Rename the header to something like `Configuring the containers using docker-compose.override.yml`
* Move this section below the `Initial Setup` and `Restore database` sections

Reply:
Will use all three suggestions